package.json

```diff
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "lint": "eslint .",
+        "flow": "flow"
+    },
+    "devDependencies": {
+        "babel-eslint": "^7.2.3",
+        "eslint": "^4.0.0",
+        "eslint-plugin-class-property": "^1.0.6",
+        "eslint-plugin-flowtype": "^2.34.0",
+        "eslint-plugin-react": "^7.0.1",
+        "flow-bin": "^0.48.0"
```

Comment:
Flow is making some kind of problems here, which I am avoiding by executing it at the last position of the travis script. If it is run before the `npm install` for all bundles it will not find some of the modules (that's because node checks the `node_modules` folder of every level).

Either we leave the check at the last position or we have to also add each bundle as `file:` dependency in here...